package ch02.note

fun main(args: Array<String>) {
    var v = try {
        5
    } catch (e: Exception) {
        null
    }
}